### DISCUSSION TITLE
Push latest commits to NuGet

### DISCUSSION THREAD

**Discussion by steve85** [ubisoft/NGitLab#525] [General]
Can one of the code owners please publish the latest changes to NuGet? I require some of the changes that were made since the 6.39.0 release to be available via NuGet. If you could do this as soon as possible it would be greatly appreciated.

Thanks!

**Comment by meziantou** [comment]
Hi,
It should be available when this pipeline succeeds https://github.com/ubisoft/NGitLab/actions/runs/6160705358

**Reply by steve85** [reply]
Thank you @meziantou!
